src/SQLite.cs

```diff
@@ -2531,19 +2563,20 @@ public Column FindColumnWithPropertyName (string propertyName)
 
 		public Column FindColumn (string columnName)
 		{
+			if(Method != MapMethod.ByName)
+				throw new InvalidOperationException($"This {nameof(TableMapping)} is not mapped by name, but {Method}.");
+
 			var exact = Columns.FirstOrDefault (c => c.Name.ToLower () == columnName.ToLower ());
 			return exact;
 		}
 
 		public class Column
 		{
-			PropertyInfo _prop;
+			MemberInfo _member;
 
 			public string Name { get; private set; }
 
-			public PropertyInfo PropertyInfo => _prop;
```

Comment:
I don't want to break public API. So please keep this property but now it can return null. Something like `=> _member as PropertyInfo`.

Reply:
Added below